src/main/java/Command.java

```diff
@@ -0,0 +1,50 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Deals with executing the task given by user input
+ */
+public abstract class Command {
+
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage) throws HenryException;
+
+    public boolean isExit(){
```

Comment:
Shouldn't there be a space between before the opening bracket?
```suggestion
    public boolean isExit() {
```